File: week_02/codewars/sylee/200903_Square_Every_Digit.py
# 1
def square_digits(num):
    return int("".join(str(int(x) ** 2) for x in str(num)))

#2
def square_digits(num):
    return int("".join(str(x ** 2) for x in map(int, str(num))))

#3
def square_digits(num):
    list_square = [int(x) ** 2 for x in str(num)]
    return int(("{}" * len(list_square)).format(*list_square))

File: week_02/codewars/chSung/200903_Square_every_digit.java
public class SquareDigit {

    public int squareDigits(int num) {
        String square_num = "";

        while (num != 0) {
            int digit = num % 10;
            square_num = digit * digit + square_num;
            num /= 10;
        }

        return Integer.parseInt(square_num);
    }

}